src/test/scala/wdl4s/WorkflowImportsSpec.scala

```diff
@@ -0,0 +1,245 @@
+      |
+      |}
+      |task sleep2 {
+      |  command { sleep 2 }
+      |}
+      |
+      |task sleep3 {
+      |  command { sleep 3 }
+      |}
+    """.stripMargin
+
+  val threestepWdl =
+    s"""
+      |import "${echoHelloWdlFile}" as funEcho
+      |
+      """.stripMargin +
+    """
+      |task ps {
+      |  command {
+      |    ps
+      |  }
+      |  output {
+      |    File procs = stdout()
+      |  }
+      |}
+      |
+      |task cgrep {
+      |  String pattern
+      |  File in_file
+      |  command {
+      |    grep '${pattern}' ${in_file} | wc -l
+      |  }
+      |  output {
+      |    Int count = read_int(stdout())
+      |  }
+      |  runtime {docker: "ubuntu:latest"}
+      |}
+      |
+      |task wc {
+      |  File in_file
+      |  command {
+      |    cat ${in_file} | wc -l
+      |  }
+      |  output {
+      |    Int count = read_int(stdout())
+      |  }
+      |  runtime {docker: "ubuntu:latest"}
+      |}
+      |
+      |workflow three_step {
+      |  call ps
+      |  call cgrep { input: in_file=ps.procs }
+      |  call wc { input: in_file=ps.procs }
+      |}
+    """.stripMargin
+
+  val basicWdlImportFile = addAndGetFile("basic", basicWdl)
+  val threeStepWdlImportFile = addAndGetFile("threestep", threestepWdl)
+  val snoozeWdlImportFile = addAndGetFile("snooze", snoozeWdl)
+  val printNumsWdlImportFile = addAndGetFile("printNums", printNumsWdl)
+
+  def noExtension(fileName: String) = fileName.replace(".wdl","")
+
+
+  val imports =
+    s"""
+       |import "$basicWdlImportFile"
+       |import "$threeStepWdlImportFile" as classicThreeStep
+       |import "$snoozeWdlImportFile" as trySleep
+       |import "$printNumsWdlImportFile"
+       |
+    """.stripMargin
+
+  val primaryWorkflow =
+    s"""
+    |
+    |task testCaseTask {
+    |  command {
+    |     echo "Ruchi's birthday: 01/19"
+    |  }
+    |}
+    |
+    |workflow testCases {
+    |  call ${noExtension(basicWdlImportFile)}.ls as ls1
+    |  call ${noExtension(printNumsWdlImportFile)}.ls as ls2
+    |
+    |  #call ${noExtension(basicWdlImportFile)}.pwd as soBasic
+    |
+    |  #call ${noExtension(printNumsWdlImportFile)}.print1
+    |  #call ${noExtension(printNumsWdlImportFile)}.print2 as printingFun
+    |
+    |
+    |  call classicThreeStep.ps
+    |  call classicThreeStep.ps as psAgain
+    |
+    |  call trySleep.sleep
+    |  call trySleep.sleep2 as sleepMore
+    |
+    |  call ${noExtension(basicWdlImportFile)}.${noExtension(echoHelloWdlFile)}.inFrench
+    |  call classicThreeStep.funEcho.inSpanish
+    |  call classicThreeStep.funEcho.inSpanish as inPortugese
+    |  call ${noExtension(printNumsWdlImportFile)}.multilingualEcho.inItalian
+    |
+    |}
+    |""".stripMargin
+
+  val wdlWithImports = imports + primaryWorkflow
+
+  val namespace = WdlNamespaceWithWorkflow.load(wdlWithImports, wdlDirectory)
+
+  "WDL file with imports" should "Have 0 tasks (3 tasks are in separate namespace)" in {
```

Comment:
Since the tests rely on creation of temp directories and files,  I was unable to move to the tests to the top. I didn't find anything unused, but let me know if you spot something specific? @mcovarr maybe its worth adding the compiler flag related to unused vals/methods in WDL4s as well, I've really found it useful.